handler.go

```diff
@@ -46,13 +68,17 @@ func (g *gzipHandler) Handle(c *gin.Context) {
 	}
 
 	gz := g.gzPool.Get().(*gzip.Writer)
-	defer g.gzPool.Put(gz)
```

Comment:
I thought about it - we are using pool of writers as optimization.

This optimization make sense it we write huge amount of compressed data - but we do it only for one endpoint - that we call every 10 seconds

What if we just create a new write all the time? It will be eventually removed later by GC.

WDYT?

Reply:
@gburanov also makes sense. Could bring an impact on high request load but lets try.